accounting/localizations/mexico.rst

```diff
@@ -2,233 +2,348 @@
 
-This tag is used to set the tax type code, transferred or withhold, applicable
-to the concept in the CFDI.
-So, if the tax is a sale tax the "Tag" field should be "IVA", "ISR" or "IEPS".
+To use the mexican invoicing you just need to do a normal invoice following the normal Odoo's behaviour. Once you validate your first
+invoice a correctly signed invoice should look like this:
 
-.. image:: media/mexico33.png
+.. image:: media/mexico56.png
    :align: center
 
-Note that the default taxes already has a tag assigned, but when you create a
-new tax you should choose a tag.
+All the marked fields in the image represent the important fields that are only relevant for Mexico and we asume you understand their
+meaning by checking and reading the `Anexo 20`_.
 
+You can send the invoice inmediatly to your customer and automatically generate the PDF format and send it to your customer, for
+that moment your invoice should look like this..
 
-Usage and testing
-=================
+.. image:: media/mexico57.png
+   :align: center
+
+Pay your invoice
+----------------
 
-Invoicing
----------
+There are 3 ways to generate a payment complement, using the same payment workflow in odoo standard.
 
-To use the mexican invoicing you just need to do a normal invoice following
-the normal Odoo's behaviour.
+a. From the invoice document.
+b. Payment first and then reconcile with invoices.
+c. From the bank statements (Once you reconcile a bank statement line and this does not have a payment yet done from a or b).
 
-Once you validate your first invoice a correctly signed invoice should look
-like this:
+**a.** From the invoice document.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
 
+Create a payment from the invoices that are related, either going to the form view and clicking the **Register Payment** button and from 
+the wizard in tha list view of invoices.
 
-.. image:: media/mexico07.png
+.. image:: media/mexico58.png
    :align: center
 
-You can generate the PDF just clicking on the Print button on the invoice or
-sending it by email following the normal process on odoo to send your invoice
-by email.
+.. image:: media/mexico59.png
+   :align: center
 
-.. image:: media/mexico08.png
+It will open the wizard to set the proper values, in case of Mexico it is important you set the proper **Payment Way** you are 
+receiving the payment.
+
+.. image:: media/mexico60.png
    :align: center
 
-Once you send the electronic invoice by email this is the way it should looks
-like.
+Your payment will be automatically signed (you can look for the payment going to the invoice and opening the payment related) if the invoice(s)
+are PUE the signature will not be done but it can be forced from the payment itself.
 
-.. image:: media/mexico09.png
+.. image:: media/mexico61.png
+   :align: center
+
+.. image:: media/mexico62.png
    :align: center
 
+**b.** Payment first and then reconcile with invoices.
+~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~~
+
+TODO: CONTINUE THIS
```

Comment:
@nhomar I think that we should only specify that this option uses the standard process.